cmd/result_object_test.go

```diff
@@ -85,74 +22,25 @@ func TestResultObjectCmd(t *testing.T) {
 			cmds:  "coreo result object",
 			desc:  "Show violating objects",
 			flags: []string{},
-			resp: []*client.ResultObject{
-				mockObject(
-					"fake-id1",
-					client.Info{
-						SuggestedAction:          "fake-suggestion",
-						Link:                     "fake-link",
-						Description:              "fake description",
-						DisplayName:              "fake-display-name",
-						Level:                    "um",
-						Service:                  "kms",
-						RuleName:                 "fake-name",
-						IncludeViolationsInCount: true,
-						TimeStamp:                "2018-10-11T17:21:55.111+00:00",
-					},
-					client.TeamInfo{
-						Name: "fake-team-name",
-						ID:   "fake-team-id",
-					}, "us-east-1"),
-			},
-			num:  mockInt(200),
-			xout: kmsKeyRotatesObjectOutput,
+			xout: "",
 		},
 		{
 			cmds:  "coreo result object",
 			desc:  "Show violating objects",
 			flags: []string{},
-			resp: []*client.ResultObject{
-				mockObject(
-					"fake-id2",
-					client.Info{
-						SuggestedAction:          "fake-suggestion",
-						Link:                     "fake-link",
-						Description:              "fake description",
-						DisplayName:              "fake-display-name",
-						Level:                    "Medium",
-						Service:                  "iam",
-						RuleName:                 "fake-name",
-						IncludeViolationsInCount: true,
-						TimeStamp:                "2018-10-11T17:21:54.448+00:00",
-					},
-					client.TeamInfo{
-						Name: "fake-team-name",
-						ID:   "fake-team-id",
-					}, "us-west-2"),
-			},
-			num:  mockInt(100),
-			xout: iamInactiveKeyNoRotationObjectOutput,
+			xout: "",
 		},
 	}
 
 	var buf bytes.Buffer
 	for _, test := range tests {
-		objectWrapper := &client.ResultObjectWrapper{
-			Objects:    test.resp,
-			TotalItems: *test.num,
-		}
-		frc := &fakeReleaseClient{objects: objectWrapper}
+		frc := &fakeReleaseClient{}
 		if test.err {
 			frc.err = errors.New("Error")
 		}
 
 		cmd := newResultObjectCmd(frc, &buf)
 		cmd.ParseFlags(test.flags)
-		err := cmd.RunE(cmd, test.args)
-
-		if (err != nil) != test.err {
-			t.Errorf("%q. expected error, got '%v'", test.desc, err)
-		}
 
 		if buf.String() != test.xout {
```

Comment:
Here an empty buffer was declared and then test whether it equals to empty string. What we should do is to call cmd.Run() and then test whether the output equals to "Findings results are deprecated, please follow the link to swagger API doc `https://api.securestate.vmware.com` \n"